packages/hash/api/.eslintrc.cjs

```diff
@@ -1,18 +1,13 @@
 /** @type {import("eslint").Linter.Config} */
 module.exports = {
-  root: true,
-  extends: ["@local/eslint-config/legacy-base-to-refactor.cjs"],
-  parserOptions: {
-    tsconfigRootDir: __dirname,
-    project: ["tsconfig.json"],
-  },
-  env: {
-    node: true,
-  },
+  ...require("@local/eslint-config/generate-workspace-config.cjs")(__dirname),
   rules: {
     // @todo Re-enable these rules once ESLint config is refactored
     "@typescript-eslint/restrict-plus-operands": "off",
     "@typescript-eslint/prefer-nullish-coalescing": "off",
   },
-  ignorePatterns: ["src/collab/**/*"],
+  ignorePatterns: [
+    ...require("@local/eslint-config/generate-ignore-patterns.cjs")(__dirname),
```

Comment:
Not adding this require would override default ignore patterns from `@local/eslint-config/generate-workspace-config.cjs`.